### DISCUSSION TITLE
Recover a repository

### DISCUSSION THREAD

**Discussion by Logitech-byte** [community/community#38101] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi. 
I want to recover the latest repository of this file if possible.
HyperScorpio/store1

Thank you.

**Comment by david-engelmann** [comment, marked as answer]
You can try and access the file from the [wayback machine](https://web.archive.org/web/20220000000000*/https://github.com/HyperScorpio)

**Reply by Logitech-byte** [reply]
Thanks, my problem as been solved, a friend still have the backup.

**Comment by alekdev0** [comment]
Hello
I want to recovery deleted 1~2months ago
I can not find deleted repo in setting/repositories/deleted repositories.
